app/src/auth/components/AuthStep/index.jsx

```diff
@@ -40,11 +41,15 @@ class AuthStep extends React.Component<Props> {
     static styles = styles
 
     componentDidMount() {
-        const { form } = this
+        const { form, props: { autoSubmitOnMount } } = this
 
         if (form) {
             form.addEventListener('change', this.onFieldChange)
         }
+
+        if (autoSubmitOnMount) {
```

Comment:
Is this flag necessary? It's only used in one place.

Reply:
This is why I used it. I want to use the regular flow, via AuthStep, but it doesn't come with the auto-submit funcionality. It's more of an enhancement, really.

I think at this point the auth flow could be refactored a little bit so this wouldn't be an edge case (which it is). Will address that next time I'm working on it, unless someone does it first.